src/lib/select/select.scss

```diff
@@ -22,17 +22,28 @@ $mat-select-panel-max-height: 256px !default;
   min-width: $mat-select-trigger-min-width;
   cursor: pointer;
   position: relative;
-  box-sizing: border-box;
 
   [aria-disabled='true'] & {
-    @include mat-control-disabled-underline();
-    border-bottom: transparent;
     background-position: 0 bottom;
     cursor: default;
     user-select: none;
   }
 }
 
+.mat-select-border {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
```

Comment:
Almost, I want to have it sink to the bottom of the container so it's missing the `top: 0`